Sources/Widgets/Widgets3D/PolyLineWidget/index.js

```diff
@@ -38,9 +38,21 @@ function vtkPolyLineWidget(publicAPI, model) {
       case ViewTypes.VOLUME:
       default:
         return [
-          { builder: vtkSphereHandleRepresentation, labels: ['handles'] },
+          {
+            builder: vtkSphereHandleRepresentation,
+            labels: ['handles'],
+            initialValues: {
+              scaleByDisplay: true,
+            },
+          },
+          {
+            builder: vtkSphereHandleRepresentation,
+            labels: ['moveHandle'],
+            initialValues: {
+              scaleByDisplay: true,
+            },
+          },
```

Comment:
@floryst could this have been shorten to this?
```js
          {
            builder: vtkSphereHandleRepresentation,
            labels: ['handles', 'moveHandle'],
            initialValues: {
              scaleByDisplay: true,
            },
          },
```

Reply:
Yeah, good point. I'll refactor that at some other time.